index.css

```diff
@@ -1,466 +1,709 @@
+  width: 95%;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--text-color);
+  text-align: justify;
+}
+
+.letsconnect {
+  margin: 4% 0 1% 0;
+  font-family: "poppins", sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 500;
+}
+
+.icons {
+  color: var(--icon-color);
+  font-size: 20px;
+  padding-top: 0.2em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+}
+
+.light-purple {
+  color: var(--light-purple);
+  font-weight: 500;
+}
+
+.h-text {
+  font-family: "poppins", sans-serif;
+  font-weight: 700;
+  width: 295px;
+  font-size: 32px;
+  color: #172b4d;
+  line-height: 44px;
+  margin-top: 12px;
+  margin-bottom: 50px !important;
+}
+
+.h-text1 {
+  font-size: 40px;
+  font-family: "poppins", sans-serif;
+  font-weight: 700;
+  width: 295px;
+  color: #172b4d;
+  line-height: 44px;
+  margin-top: 12px;
+}
+
+.info-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.canopy {
+  font-family: "poppins", sans-serif;
+  font-weight: 600;
+  width: 56px;
+  height: 13px;
+  font-size: 13px;
+  font-style: normal;
+  color: #344563;
+  line-height: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.h-text-name {
+  font-family: "poppins", sans-serif;
+  width: 295px;
+  height: 80px;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  align-items: center;
+  padding-left: 24px;
+}
+
+li {
+  list-style: none;
+}
+
+.portfolio {
+  margin: 44px auto 44px auto;
+  padding: 0 1em;
+  width: 375px;
+  height: 596px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2em;
+}
+
+.project-cards-desktop {
+  display: none;
+}
+
+.test {
+  margin-top: 114px;
+}
+
+.test2 {
+  margin-bottom: 0;
+  padding-bottom: 44px;
+}
+
+.project {
+  width: fit-content;
+  max-width: max-content;
+  padding: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  background: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.skills-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+}
+
+.checkbox {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.checkbox > i {
+  font-size: 0.5rem;
+  color: lightgray;
+  margin: 0 1em;
+}
+
+.tech-list {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 0.25em 0;
+}
+
+.project-tech {
+  background-color: rgba(127, 140, 255, 0.5);
+  padding: 0.25em 0.5em;
+  border-radius: 0.5em;
+  color: var(--bold-purple);
+  margin: 0.5em 0.35em 0.5em 0;
+}
```

Comment:
Kindly fix the color of project areas like on the Figma template. Check the color codes below 👍 
<img width="737" alt="Screenshot 2023-02-25 at 8 57 58 PM" src="https://user-images.githubusercontent.com/25887253/221373544-2f895949-9f15-4b56-89bc-7e13681e98b0.png">

<img width="737" alt="Screenshot 2023-02-25 at 8 57 29 PM" src="https://user-images.githubusercontent.com/25887253/221373542-8871f425-3c39-4c0d-9a8c-b5dba2d0a9f7.png">
<img width="737" alt="Screenshot 2023-02-25 at 8 57 21 PM" src="https://user-images.githubusercontent.com/25887253/221373540-dc710c1b-5f53-4118-bbd0-e76d9a0c2d0a.png">